src/cdk-experimental/listbox/listbox.ts

```diff
@@ -355,7 +390,10 @@ export class CdkListbox<T> implements AfterContentInit, OnDestroy, OnInit, Contr
     }
 
     /** Will select an option if shift was pressed while navigating to the option */
-    const isArrow = (keyCode === UP_ARROW || keyCode === DOWN_ARROW);
+    const isArrow = (keyCode === UP_ARROW
+        || keyCode === DOWN_ARROW
+        || keyCode === LEFT_ARROW
+        || keyCode === RIGHT_ARROW);
```

Comment:
Should this always be all four, or should it depend on the orientation?

Reply:
The isArrow variable is true if any arrow key is pressed. In the following if statement, a selection change will only occur if there is a new active option which the listKeyManager keeps track of. The listKeyManager knows the orientation, so if RIGHT is pressed while I vertical mode, the active item won't switch and even though isArrow is true no action is taken.